### PR TITLE
text banner complete

### LINKED ISSUE #42: LogoStrip
image

File is at `src/components/clamps/LogoStrip.jsx`

Images are in the `public/images/lineclamps/companies`

Use Image component from next.

### PR DESCRIPTION
**Pull request by pjflipper1**

### CODE REVIEW

File: src/components/clamps/LogoStrip.jsx [hunk 1]
```diff
@@ -1,7 +1,29 @@
 import React from "react";
+import Image from "next/image";
 
 const LogoStrip = () => {
-  return <div>LogoStrip</div>;
+  return (
+    <div className="flex justify-between p-4">
```
Comment (SafeDuck) [inline src/components/clamps/LogoStrip.jsx:6]: have width `w-1/2`

File: src/components/clamps/LogoStrip.jsx [hunk 2]
```diff
@@ -1,7 +1,29 @@
...
+      <Image
+        src="/images/lineclamps/companies/NASLogo.webp"
```
Comment (SafeDuck) [inline src/components/clamps/LogoStrip.jsx:8]: static import images at the top

File: src/components/clamps/LogoStrip.jsx [hunk 3]
```diff
@@ -1,7 +1,29 @@
...
+        alt="SAE Logo"
+        width={180}
```
Comment (SafeDuck) [inline src/components/clamps/LogoStrip.jsx:10]: have width and height as taliwind utility classes

**Review by SafeDuck — CHANGES_REQUESTED** [review]

**Comment by prodbysuj** [comment]
352362590-3c0450ce-c0ba-4ef2-9d51-61c31d5e1302

**Pull request closed without merging**